public/styles/nav.css

```diff
@@ -0,0 +1,37 @@
+.container-nav-bar {
```

Comment:
we should make a decision on how we are going to handle styling. If it should be centrally located or if we what it linked per file. Either way it should be stored in the src folder not public.